docs/development-setup.md

```
@@ -0,0 +1,419 @@
+    "containerization-assist-dev": {
+      "command": "npx",
+      "args": ["tsx", "watch", "./src/cli/cli.ts"],
+      "env": {
+        "MCP_MODE": "true",
+        "MCP_QUIET": "true",
+        "NODE_ENV": "development"
+      }
+    }
+  }
+}
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)